ini/stm32f4.ini

```diff
@@ -243,6 +243,15 @@ build_flags       = ${stm_flash_drive.build_flags}
   -DUSE_USBHOST_HS -DUSE_USB_HS_IN_FS -DUSBD_IRQ_PRIO=5 -DUSBD_IRQ_SUBPRIO=6
   -DHSE_VALUE=8000000U -DHAL_SD_MODULE_ENABLED
 
+#
+# BigTreeTech SKR V2.0 (STM32F407VGT6 ARM Cortex-M4) with USB Media Share Support
+#
+[env:BIGTREE_SKR_2_USB]
+extends           = env:BIGTREE_SKR_2
```

Comment:
Note: The PlatformIO IDE extension for VSCode requires a `platform` field or it won't display the environment.